### ISSUE TITLE
Mixed line: rubric and default text

### ISSUE THREAD

**Issue by matgille** [SegmOnto/Guidelines#42]
In the example below, should we split the line containing the rubric in two  and have a DefaultLine and a HeadingLine ? If not, why ? 

![Capture d’écran du 2021-12-09 16-19-45](https://user-images.githubusercontent.com/74190212/145423766-ec0043b0-669a-441e-92cf-854b41afa3ad.png)

**Comment by Jean-Baptiste-Camps** [comment]
Yes, we should. It mixes two "layout' lines, whose content is of different nature.

**Comment by matgille** [comment]
Ok, thanks; then maybe there is a need to precise it in the examples, with a caption for instance ? Or to precise the definition of the line (not defined by the text between two linebreaks).

**Comment by gabays** [comment]
I agree: can we use your examples in our guidelines? If yes could you provide the shelfmark of the ms?

**Comment by matgille** [comment]
U/Bc Ms 251, Universidad de Valladolid. https://uvadoc.uva.es/handle/10324/409
